.github/workflows/AWS-auth.yml

```diff
@@ -0,0 +1,68 @@
+name: AWS Auth + Welcome Email
+
+on:
+  push:
+    branches:
+      - 'test_userpool_creation'
+    paths:
+        '**.json'
```

Comment:
Not a blocker, but I think this should really be `config/**.json` just in case we end up with json files elsewhere.